src/pages/projectSettings/tabs/PeriodicBuildsTab/getFormSchema.ts

```diff
@@ -0,0 +1,126 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { timeZones } from "constants/fieldMaps";
+import { form, ProjectType } from "../utils";
+
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
+  projectType: ProjectType,
+  timezone: string
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    description: "Configure tasks to run at fixed intervals.",
+    ...overrideRadioBox(
+      "periodicBuilds",
+      ["Override Repo Commands", "Default to Repo Commands"],
+      {
+        type: "array" as "array",
+        default: [],
+        items: {
+          type: "object" as "object",
+          properties: {
+            intervalHours: {
+              type: "number" as "number",
+              title: "Interval",
+            },
+            configFile: {
+              type: "string" as "string",
+              title: "Config File",
+              minLength: 1,
+              default: "",
+            },
+            alias: {
+              type: "string" as "string",
+              title: "Alias",
+              default: "",
+            },
+            message: {
+              type: "string" as "string",
+              title: "Message",
+              default: "",
+            },
+            timezone: {
+              type: "string" as "string",
+              title: "Time Zone",
+              default: timezone,
+              oneOf: timeZones.map(({ str, value }) => ({
+                type: "string" as "string",
+                title: str,
+                enum: [value],
+              })),
+            },
+            nextRunTime: {
+              type: "string" as "string",
+              title: "Next Run Time",
```

Comment:
This time picker isn't working as I'd expect. Every time I click "OK", it translates my time. For example, if I try to click OK on 23:00 for 4/8
<img width="399" alt="image" src="https://user-images.githubusercontent.com/26798134/161626066-97cfbc82-79b3-471d-bf5c-c4b23381742b.png">
it gives me 3 on 4/9: 
<img width="316" alt="image" src="https://user-images.githubusercontent.com/26798134/161626136-2214bfc6-a9bb-4b5d-9eba-9710affb41ab.png">